src/content/docs/en/recipes/add-yaml-support.mdx

```diff
@@ -34,3 +34,15 @@ Astro builds on top of Vite, and supports both Vite and Rollup plugins. This rec
     ```js
     import yml from './data.yml';
     ```
+
+    :::note
+    While you can now import YAML data in your Astro project, your editor will not provide types for the imported data. To add types, create a `*.d.ts` file in your project and add the following:
+    ```ts
+    // specify the file extension you want to import
+    declare module "*.yml" {
+      const value: any; // Add better type definitions here if desired
+      export default value;
+    }
+    ```
```

Comment:
Nit: uppercase first letter for both comments.

Also, I think we might want to add a code block title here to specificy **where** in your project the `*.d.ts` file goes.
````suggestion
    ```ts
    // Specify the file extension you want to import
    declare module "*.yml" {
      const value: any; // Add better type definitions here if desired
      export default value;
    }
    ```
````

Reply:
*Appended the note to state you should place it in the `src` directory